### ISSUE TITLE
The javabean you created must implements Serializable

### ISSUE THREAD

**Issue by cenumi** [mancj/MaterialSearchBar#44]
The javaBean you created must implements Serializable or when you start another activity,
"Java.lang.RuntimeException: Parcel: unable to marshal value com.xxx.xxx.yourJavaBean@xxxx1234"
shows.

**Comment by jld3103** [comment]
Same error here

**Comment by ivan-avalos** [comment]
Yes, me toooooooooooooooooooooooooooooooo!!!! This project seems to be discontinued, let's use another!

**Comment by ivan-avalos** [comment]
I found a partial solution to this problem: 
```
searchBar.isSaveEnabled = false
searchBar.isSaveFromParentEnabled = false
```
I don't know what is the second for, but it worked for me!
